multicluster/controllers/multicluster/member/clusterset_controller.go

```diff
@@ -109,6 +109,12 @@ func (r *MemberClusterSetReconciler) Reconcile(ctx context.Context, req ctrl.Req
 			r.clusterID = common.InvalidClusterID
 			r.clusterSetID = common.InvalidClusterSetID
 		}
+		if r.remoteCommonArea == nil {
```

Comment:
Question - when a ClusterSet is recreated, do we reconcile all these resources, remove stale ones and recreate missing ones? Could that have synchronization issues with the cleanup process?

Reply:
There is another PR https://github.com/antrea-io/antrea/pull/5410 to recreate the resources when the ClusterSet is created again. There might be synchronization issues, I am still checking and verifying it.